msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/broker/brokerapi/TestCase1561136.java

```diff
@@ -51,6 +52,7 @@
 @SupportedBrokers(brokers = BrokerHost.class)
 @RetryOnFailure(retryCount = 2)
 @LocalBrokerHostDebugUiTest
+@Ignore("Seeing an error related to encryption at the 'getAccounts()' steps")
```

Comment:
Interesting...we should investigate this

Reply:
Me and Dome investigated, but didn't find anything conclusive besides this not repro-ing during a manual run, he doesn't have much bandwidth at this time so we're going to look into the ui test case after the release.